src/bundle/Resources/public/scss/_custom.scss

```diff
@@ -289,21 +289,21 @@ $pagination-disabled-bg: transparent;
 $pagination-disabled-border-color: transparent;
 
 // Modals
-$modal-header-padding-y: 16px;
+$modal-header-padding-y: calculateRem(16px);
 $modal-header-padding-x: 0;
-$modal-header-border-width: 1px;
+$modal-header-border-width: calculateRem(1px);
 $modal-header-border-color: $ibexa-color-light-500;
-$modal-title-font-size: 28px;
+$modal-title-font-size: calculateRem(28px);
 $modal-content-border-width: 0;
 $modal-content-border-radius: $ibexa-border-radius;
-$modal-inner-padding: 24px 0;
```

Comment:
Note: change explained in https://github.com/ibexa/admin-ui/pull/335#discussion_r811405258